src/lib.rs.in

```diff
@@ -583,6 +600,18 @@ class DocumentAccessResult {
 
 });
 
+foreigner_class!(
+class IronSdkAdvanced {
+    self_type IronSdkAdvanced<'a>;
+    private constructor = empty;
+    /// Encrypt the provided document bytes. Return the encrypted document encryption keys (EDEKs) instead of creating a document entry in the IronCore webservice.
+    /// 
+    /// @param a0 `documentData`   - bytes of the document to encrypt
+    /// @param a1 `encryptOpts`    - optional document encrypt parameters
+    /// @param a2 `id`             - unique id to use for the document. Document id will be stored unencrypted and must be unique per segment
```

Comment:
This param doesn't exist here.

Reply:
That's fair. So then I take it I should also remove it from document_encrypt?
https://github.com/IronCoreLabs/ironoxide-java/blob/137b26ade1671508e522517505198e02531257d1/src/lib.rs.in#L655-L658

Reply:
Yep.

Reply:
Done